web/app/dao/DatasetsDAO.java

```diff
@@ -2233,21 +2234,29 @@ public static DatasetSecurity getDatasetSecurityByDatasetUrn(String datasetUrn)
 
 	private static DatasetSecurity mapToDatasetSecurity(Map<String, Object> result) throws IOException {
 		ObjectMapper om = new ObjectMapper();
+		Map<String, Object> datasetClassification =
+				StringUtils.isBlank((String) result.get("dataset_classification")) ? null
```

Comment:
Thanks for suggesting 'getOrDefault'. But in this case, the map does contain this key, it's just the value could be null or empty so the JSON mapping will throw error.